src/providers/export/tensorFlowPascalVOC.test.ts

```diff
@@ -118,14 +131,14 @@ describe("TFPascalVOC Json Export Provider", () => {
             expect(createContainerCalls[4][0].endsWith("/ImageSets/Main")).toEqual(true);
 
             const writeBinaryCalls = storageProviderMock.mock.instances[0].writeBinary.mock.calls;
-            expect(writeBinaryCalls.length).toEqual(4);
+            expect(writeBinaryCalls.length).toEqual(testAssets.length);
             expect(writeBinaryCalls[0][0].endsWith("/JPEGImages/Asset 1.jpg")).toEqual(true);
             expect(writeBinaryCalls[1][0].endsWith("/JPEGImages/Asset 2.jpg")).toEqual(true);
             expect(writeBinaryCalls[2][0].endsWith("/JPEGImages/Asset 3.jpg")).toEqual(true);
             expect(writeBinaryCalls[3][0].endsWith("/JPEGImages/Asset 4.jpg")).toEqual(true);
 
             const writeTextFileCalls = storageProviderMock.mock.instances[0].writeText.mock.calls as any[];
-            expect(writeTextFileCalls.length).toEqual(11);
+            expect(writeTextFileCalls.length).toEqual(17);
```

Comment:
Any way to calculate this number from other variables? Or document why it's 17? Just to make the test a little less fragile if things change

Reply:
Totally agree with you, was able to figure out the computation.

Reply:
We could remove this line.  That is the total number of text files exported that for pascal voc is something like 1 pbtxt + 1 * tag * 2 (train/validation) + 1 per asset.  But we test the presence of these specific files few lines below in same test so we could remove it.